terra-notebooks-playground/Hail - How to run a notebook in the background.ipynb

```diff
@@ -0,0 +1,224 @@
+    "</div>\n",
+    "\n",
+    "How to use this notebook:\n",
+    "1. Copy this notebook to the workspace that contains the long-running notebook you wish to run in the background.\n",
+    "1. Edit the filename in `NOTEBOOK_TO_RUN` to be the name of the notebook in this workspace you wish to run.\n",
+    "1. Use menu `Cell -> Run All` to tell the kernel to run this notebook to completion.\n",
+    "1. Close this tab and do other work. (note: when you close the tab, the outputs in *this* notebook will no longer be updated, and that is fine, because what we really want are the outputs in the Hail notebook.)\n",
+    "1. When the Hail job is complete:\n",
+    "    1. Your Hail cluster will pause (if you have no other notebooks open).\n",
+    "    1. You will see a copy of your notebook in the notebooks tab with a date and timestamp suffix --> this is the output file.\n",
+    "    1. You will also see the Hail log in `gs://<workspace bucket>/hail-logs/YYYYMMDD/hail*.log`\n",
+    "\n",
+    "If you check back, and you do not see the output notebook, the Hail job is still running. To confirm this, first notice whether your Cloud analysis environment is still running. If it is, from the terminal run `yarn application -list` to see that a spark context is still processing data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import nbformat\n",
+    "from nbconvert.preprocessors import CellExecutionError\n",
+    "from nbconvert.preprocessors import ExecutePreprocessor\n",
+    "\n",
+    "import os\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "<b>Change the cell below</b> to hold the name of the notebook in this workspace that you wish to run in the background. There is no need to modify any of the other cells.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#---------[ CHANGE THIS TO BE THE NAME OF THE NOTEBOOK YOU WISH TO RUN ]-----------\n",
+    "NOTEBOOK_TO_RUN = 'your_notebook.ipynb'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Programmatically set output paths"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Formulate a new filename for the output notebook so that it include a date and timestamp for when it was executed."
```

Comment:
"so that it **includes**"
's' at the end of includes

Reply:
fixed that typo, thanks!